### PR TITLE
PR-1-Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by ivanmvh**
### Description
This project will lay foundations for our Bookstore website. We will create the React part of the app. We will also set up routing using React Router.

### General requirements

- [x] Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
- [x] Make sure that we used the correct flow [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).
- [x] Make sure that we documented our work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
- [x] HTML/CSS requirements
- [x] Follow our list of the [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
- [x] JavaScript requirements
- [x] Follow our list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).

### Project requirements

- [x] Initialise React app.
- [x] The building blocks of our app should be set as reusable components. Create a directory for our components.
- [x] Add [React Router](https://reactrouter.com/web/guides/quick-start) and set two <Route>s and <Link>s for the app's navigation:
- [x] Books - the default view:
- [x] Should display the list of books (empty at this point but it should be ready for the data) with the Remove button (no functionality yet).
- [x] Create a component called Book for displaying a single book (receive a title and an author as a prop) and reuse it in a component that displays a list of books.
- [x] Should display a form for adding a book (no functionality yet).
- [x] Create a separate component for this form (with inputs for a title and an author [IMPORTANT in the design we can see an input for a category - please replace it with an author.]).
- [x] Categories:
- [x] Should display a button "Check status" only.
- [x] Styling is not required at this point.

@ivanmvh

### CODE REVIEW

**Review by codecaiine — COMMENTED** [review]
Hi @ivanmvh , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that the PR used the wrong flow. Please make sure that you use the correct flow [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md) in this project. In Gitflow Any changes required in the application should be introduced in **feature branches** and approved before merging them to the **development branch**. Changes to the **main branch** can be merged only from the development branch and reviewed. :+1: 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/Pages/Categories.jsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const Categories = () => (
+  <h1>Check status</h1>
```
Comment (jerryowusu) [inline src/Pages/Categories.jsx]: The categories component should display a button with text `Check Status`. However, you are displaying a header tag, kindly check and fix this. :wink:

File: src/styles.css [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+.bookTitleForm {
+  width: 41.5rem;
+  height: 2.813rem;
+  margin: 1.188rem 2.125rem 0 0;
+  padding: 0.813rem 35.688rem 0.813rem 1.063rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+}
+
```
Comment (jerryowusu) [inline src/styles.css]: [OPTIONAL] Though styling is not required at this point, the styles you have applied makes it difficult to see the whole page of the app which does not make it proffessional and up to standard. Kindly check and update it. :pray:

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @ivanmvh , 

Good job so far!
Your project is looking fantastic so far :star_struck: 

However, there are few changes you need to make in order to move on to the next level :sunglasses:

almost-there-reach


## Highlights
All linters are passing :heavy_check_mark: 
Great effort so far :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Pages/Categories.jsx (hunk 1 above)
Comment (Whoistolu) [reply src/Pages/Categories.jsx]: - Well resolved @ivanmvh 👍🏽 
Screenshot 2023-01-20 at 16 46 12

**Review by Whoistolu — APPROVED** [review]
### Hi @ivanmvh 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**